### PR TITLE
Feature/country translation

### LINKED ISSUE #270: Country Name Translations
Country Name should be localized [low priority]

Screen Shot 2020-09-23 at 9 15 27 AM

### PR DESCRIPTION
**Pull request by norbertschuler**
Fixes #270

Changes in this pull request:
- added country translation files (a few keys are not available from my source https://stefangabos.github.io/world_countries/, where I just keep the English names from https://github.com/Plant-for-the-Planet-org/planet-webapp/blob/develop/src/utils/countryCurrency/countriesData.json if no translation is available)
- changed default language to English (in tenant config files)

### CODE REVIEW

File: public/tenants/planet/config.ts [hunk 1]
```diff
@@ -11,7 +11,7 @@ const config = {
     secondaryFontFamily: '"Open Sans",Helvetica,Arial,sans-serif',
     secondaryFontURL: `${process.env.CDN_URL}/media/fonts/opensans/open-sans.css?v1.0`,
   },
-  languages: ['de', 'en'],
+  languages: ['en', 'de'],
```
Comment (norbertschuler) [inline public/tenants/planet/config.ts:14]: First language in this array becomes the default language!

File: src/features/common/Layout/Footer/SelectLanguageAndCountry.tsx [hunk 2]
```diff
@@ -136,7 +129,10 @@ export default function TransitionsModal(props) {
 
 // Maps the radio buttons for countries
 function MapCountry(props) {
-  const { sortedCountriesData, value, handleChange } = props;
+  const { t } = useTranslation(['country']);
+  
+  const { value, handleChange } = props;
+  const sortedCountriesData = sortCountriesByTranslation(t);
```
Comment (norbertschuler) [inline src/features/common/Layout/Footer/SelectLanguageAndCountry.tsx:135]: This add some unnecessary sorting to the app at runtime, but I cannot do the sorting on the translated country names before i18n is initialized.

File: src/features/common/Layout/Footer/index.tsx [hunk 3]
```diff
@@ -75,6 +65,10 @@ export default function Footer() {
         countryCode = localStorage.getItem('countryCode');
         if (countryCode) setSelectedCountry(countryCode);
       }
+      if (localStorage.getItem('language')) {
+        langCode = localStorage.getItem('language');
+        if (langCode) setLanguage(langCode);
```
Comment (norbertschuler) [inline src/features/common/Layout/Footer/index.tsx:70]: Loading from local storage as other wise the language shown in the footer is the default `English` if i18n was not yet initialized.

File: public/tenants/stern/config.ts [hunk 4]
```diff
@@ -1,7 +1,7 @@
 const config = {
   tenantName: 'stern',
   tenantURL: 'baeume.stern.de',
-  languages: ['de', 'en'],
+  languages: ['en', 'de'],
```
Comment (harshvitra) [inline public/tenants/stern/config.ts:4]: We might want to keep de as default for Stern

**Review by harshvitra — APPROVED** [review]

**Review by harshvitra — APPROVED** [review]

File: public/tenants/stern/config.ts (hunk 4 above)
Comment (norbertschuler) [reply public/tenants/stern/config.ts]: In that case we would need to add some countries in `de/country.json` as they only exists in `en/country.json` for now and therefore Stern would show the placeholder value if `de` in the default country an a user has chosen the German language. But I assume the country resources will now be transferred to LingoHub and I can add German translations for the remaining country names there.

**Pull request merged**